driver/driver.c

```diff
@@ -29,7 +29,7 @@ NTSTATUS
 DriverEntry(PDRIVER_OBJECT DriverObject,
             PUNICODE_STRING RegistryPath)
 {
-    WNBD_LOG_LOUD(": Enter");
+    WNBD_LOG_LOUD("Enter");
```

Comment:
I'm wondering if we should keep those ``` WNBD_LOG_LOUD("Enter");```

Reply:
It will help in the event of stack corruption (very small chance for it to happen, please let me know if you want to keep them).